crates/nostr/src/message/relay/mod.rs

```diff
@@ -165,10 +165,10 @@ impl<'de> Deserialize<'de> for RelayMessage {
 
 impl RelayMessage {
     /// Create `EVENT` message
-    pub fn event(subscription_id: SubscriptionId, event: Event) -> Self {
+    pub fn event(subscription_id: SubscriptionId, event: &Event) -> Self {
```

Comment:
I don't like too much this (accept a ref and internally clone). I prefer to avoid it. I'm going to close this PR.

Note: in bindings (both FFi and JS) you'll see that most of args of func/methods are references and internally are cloned. But that is a particular case (in UniFFI is enforced and in JS is to avoid panics at runtime in certain conditions).